src/db/models/User.ts

```diff
@@ -206,6 +207,12 @@ UserModel.init(
             allowNull: false,
             defaultValue: {},
         },
+        locale: {
+            type: DataTypes.ENUM("en", "fr"),
```

Comment:
should we add a default value ? 
what happens when we run the migrations and existing users don't have a locale value ?

Reply:
I dont think so, it would just initialize the value to null/undefined and take the default value from the portal. So far I tested it all the way in CQDG and it didn't pose a problem. It just defaulted to the value of the portal and then once I switched it, it updated the user preferences.

Assigning a default value would be problematic because CHOP project would most likely be en while CLIN/CQDG would be french.